docs/source/asr/example_training_asr_on_kinyarwanda_mcv.rst

```diff
@@ -0,0 +1,641 @@
+########################################################################
```

Comment:
Dont put this at the root of the asr docs. Put it in some subfolder for examples. Also, why is an example in the docs? It should either be a tutorial or a script inside examples dir?